tools/actorfixer.py

```diff
@@ -344,7 +344,7 @@
     "func_800C9CEC": "SurfaceType_IsHookshotSurface",
     "func_800C9D14": "SurfaceType_IsIgnoredByEntities",
     "func_800C9D50": "SurfaceType_IsIgnoredByProjectiles",
-    "func_800C9D8C": "SurfaceType_IsConveyor",
+    "func_800C9D8C": "SurfaceType_GetConveyorType",
```

Comment:
Don't forget to add the new rename you are adding
```suggestion
    "func_800C9D8C": "SurfaceType_GetConveyorType",
    "SurfaceType_IsConveyor": "SurfaceType_GetConveyorType",
```

Reply:
Are the functions you renamed used a much? Maybe those renames should be added to actorfixer too

Reply:
This function is only used two more times in the entire game. One in `Player_UpdateCommon` which I assume you haven't gotten to yet, and in `ovl_Effect_Ss_Bubble` which is untouched and unreserved. So this function can honestly probably go from actorFixer

Reply:
In that case let's remove it lol